examples/simple-filtering/simple_filtering.bal

```diff
@@ -0,0 +1,90 @@
+import ballerina/io;
+import ballerina/runtime;
+
+// Create a record type that represents the Person.
+type Person record {
+    string name;
+    int age;
+    string status;
+    string address;
+    string phoneNo;
+};
+
+// Create a record type that represents the Child.
+type Child record {
+    string name;
+    int age;
+    string city;
+};
+
+
+int index = 0;
+
+// Streams which are based on above-created constraint types.
+stream<Person> personStream = new;
+stream<Child> childrenStream = new;
+
+Child[] globalChildrenArray = [];
+
+function initFilterQuery() {
+    // Streaming query which filter events based on the attribute age.
+    // Filterted events are pushed to a stream called `childrenStream`.
```

Comment:
// Filtered events are pushed to a stream called `childrenStream`.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/12750.